docs/reliability/innodb_corrupt_table_action.md

```diff
@@ -43,8 +43,8 @@ This feature adds a new system variable.
 | Default      | assert                |
 | Range        | assert, warn, salvage |
 
-* With the default value, `assert`, *XtraDB* will intentionally crash the server with an assertion failure as it would normally do when detecting corrupted data in a single-table tablespace.
+* With the default option value is `assert`, then *XtraDB* intentionally crashes the server with an assertion failure as it would normally do when detecting corrupted data in a single-table tablespace.
```

Comment:
With the default option value is `assert`, then *XtraDB* intentionally crashes   ->   If the default option value is `assert`, *XtraDB* intentionally crashes

Reply:
Should be a gerund to match the other two options.